source/features/hide-navigation-hover-highlight.tsx

```diff
@@ -7,7 +7,7 @@ const html = document.documentElement;
 
 function init(): void {
 	html.setAttribute(attribute, '');
-	html.addEventListener('navigation:focus', () => {
+	html.addEventListener('navigation:keydown', () => {
```

Comment:
Bonus bugfix: likewise, just hovering an item focuses the item, so I had to change it to at least `keydown` (not perfect, but not as noticeable as in `dim-bots`)